### PR TITLE
feat(access-control): Setup skeleton for editing an access

### PR DESCRIPTION
**Pull request by darrikonn**
# Setup skeleton for editing an access

## What

Skeleton for editing an access

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/settings/access-control/src/screens/Access/Access.tsx [hunk 1]
```diff
@@ -21,6 +59,76 @@ function Access() {
...
+                      defaultMessage: 'Útskýring',
+                    })}
+                  </Text>
+                </T.HeadData>
+                <T.HeadData>
+                  <Text variant="small" color="blue600">
+                    {formatMessage({
+                      id: 'service.portal:access-control-access-created',
+                      defaultMessage: 'Í gildi frá',
+                    })}
```
Comment (eirikurn) [inline libs/service-portal/settings/access-control/src/screens/Access/Access.tsx:111]: These should be namespaced separately, eg under service portal settings or access-control namespaces.

File: libs/service-portal/settings/access-control/src/screens/Access/Access.tsx [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+    scopes: [
+      {
+        id: '1337',
+        displayName: 'Undiraðgangur',
+        description: 'Útskýring',
+        validTo: '',
+      },
+    ],
+  },
+]
```
Comment (eirikurn) [inline libs/service-portal/settings/access-control/src/screens/Access/Access.tsx:33]: I assume we'll fetch groups/scopes to list, separately from the delegation which lists which scopes the user has, and how long they're valid.

Comment (darrikonn) [reply libs/service-portal/settings/access-control/src/screens/Access/Access.tsx:33]: Yeah, I just decided to stop there and work on getting the actual data in.

**Review by oddsson — APPROVED** [review]

**Review by eirikurn — APPROVED** [review]
Great!

**Pull request merged**